doc/man3/OSSL_PROVIDER.pod

```diff
@@ -154,13 +156,21 @@ Only text parameters can be given, and it's up to the provider to
 interpret them.
 
 OSSL_PROVIDER_get_conf_parameters() retrieves global configuration parameters
-associated with B<prov>.
+associated with I<prov>.
 These configuration parameters are stored for each provider by the OpenSSL core,
 not the provider itself, parameters managed by the provider are queried via
 B<OSSL_PROVIDER_get_params()> described above.
 The parameters are returned by reference, not as copies, and so the elements of
 the I<param> array must have B<OSSL_PARAM_UTF8_PTR> as their B<data_type>.
 
+OSSL_PROVIDER_conf_get_bool() parses the global configuration parameter I<name>
+associated with provider I<prov> as a boolean value, returning a default value
+I<defval> when unable to retrieve or parse the parameter.
+Parameter values equal (all uppercase also accepted) to C<1>, C<on>, C<yes>,
+or C<true> yield a true (nonzero) result.
+Parameter values equal (perhaps all uppercase) to C<0>, C<off,> C<no>, or
```

Comment:
Change _perhaps all uppercase_ to _all uppercase also accepted_ please to be consistent.

Reply:
Made the match case insensitive and adjusted doc accordingly.